models/classes/class.TestsService.php

```diff
@@ -28,17 +28,18 @@
 use oat\tao\model\service\ServiceFileStorage;
 use oat\taoTests\models\TestModel;
 use oat\taoTests\models\MissingTestmodelException;
-
+use oat\tao\model\OntologyClassService;
+    
 /**
  * Service methods to manage the Tests business models using the RDF API.
  *
  * @access public
  * @author Joel Bout, <joel.bout@tudor.lu>
  * @package taoTests
- 
+     
  */
 class taoTests_models_classes_TestsService
-    extends tao_models_classes_ClassService
+    extends OntologyClassService
```

Comment:
The `extends OntologyClassService` can be on the same line as the `class` definition